packages/fxa-payments-server/src/lib/stripe.ts

```diff
@@ -0,0 +1,287 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file contains code moved from `SubscriptionCreate/index`, which was/is pretty unwieldy, and I want separate out payment provider specific stuff when possible.